piccolo_admin/endpoints.py

```diff
@@ -416,6 +460,32 @@ def __init__(
             methods=["POST"],
         )
 
+        #######################################################################
+        # Media
+
+        private_app.add_api_route(
+            path="/media/",
+            endpoint=self.store_file,  # type: ignore
+            methods=["POST"],
+            tags=["Media"],
+            response_model=StoreFileResponseModel,
+        )
+
+        private_app.add_api_route(
+            path="/media/generate-file-url/",
+            endpoint=self.generate_file_url,  # type: ignore
+            methods=["POST"],
+            tags=["Media"],
+            response_model=GenerateFileURLResponseModel,
+        )
+
+        # TODO - this is problematic ... mount a media folder by default?
+        # Allow the user to customise the location???
+        private_app.mount(
+            path="/media-files/",
+            app=StaticFiles(directory=MEDIA_PATH),
+        )
+
```

Comment:
I don't think this is too much of a problem since even Django have the folder for media files under MEDIA_ROOT. Maybe we can just rename `example_media` to `uploads` to make it clearer what the purpose of that folder is.